src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenIterators.vb

```diff
@@ -1830,5 +1830,28 @@ BC35000: Requested operation is not available because the runtime library functi
                 </expected>)
         End Sub
 
+        <Fact>
+        <WorkItem(220696, "https://devdiv.visualstudio.com/defaultcollection/DevDiv/_workitems#_a=edit&id=220696")>
+        Public Sub IteratorPropertySetterDebugInfo()
```

Comment:
Is this test testing something more that the one in IteratorTests.vb? There is a "SetterDebugInfo" in the name of the test, but I don't see anything specific to DebugInfo. I might be missing something though.